### PR TITLE
Enabled support for Cloudfront distributions

### PR DESCRIPTION
**Pull request by ryancormack**

### CODE REVIEW

File: Watchman.AwsResources.IntegrationTests/Service/CloudFront/CloudFrontSourceTests.cs [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+using NUnit.Framework;
+using TestHelper;
+using Watchman.AwsResources.Services.CloudFront;
+
+namespace Watchman.AwsResources.IntegrationTests.Service.CloudFront
+{
+    public class CloudFrontSourceTests
+    {
+        [Test]
+        public async Task GetResourceNamesAsyncShouldReturnResults()
```
Comment (ryancormack) [inline Watchman.AwsResources.IntegrationTests/Service/CloudFront/CloudFrontSourceTests.cs]: I'm not sure of the value of this test, but they existed already, so I've added them here.

If the AWS Creds you test against don't have any Cloudfront distributions then it's going to fail

Comment (tomhaigh) [reply Watchman.AwsResources.IntegrationTests/Service/CloudFront/CloudFrontSourceTests.cs]: yeah agreed, guess it's covered by other tests anyway?

Comment (ryancormack) [reply Watchman.AwsResources.IntegrationTests/Service/CloudFront/CloudFrontSourceTests.cs]: Yeah, it's covered elsewhere

File: Watchman.Engine/Alarms/Defaults.cs [hunk 2]
```diff
@@ -793,6 +793,28 @@ public static class Defaults
...
+                EvaluationPeriods = 5,
+                Threshold = new Threshold
+                {
+                    ThresholdType = ThresholdType.Absolute,
+                    Value = 10
+                },
+                DimensionNames = new[] { "DistributionId" },
+                ComparisonOperator = ComparisonOperator.GreaterThanOrEqualToThreshold,
+                AlertOnInsufficientData = false,
+                Statistic = Statistic.Average,
```
[the remark this answers is not in the document]
Comment (ryancormack) [reply Watchman.Engine/Alarms/Defaults.cs]: AWS/Cloudfront/Cloudwatch on offer Average as the statistic for this alarm. https://docs.aws.amazon.com/AmazonCloudFront/latest/DeveloperGuide/monitoring-using-cloudwatch.html#cloudfront-metrics

**Pull request merged**